papaya-core/src/main/java/pl/edu/agh/papaya/service/sprint/SprintCreationWizard.java

```diff
@@ -19,6 +24,20 @@
         this.sprintService = Objects.requireNonNull(sprintService);
     }
 
+    public SprintCreationWizard withName(String name) {
+        if (name != null) {
```

Comment:
Chcemy żeby wywołanie `withName(null)` nie zrobiło absolutnie nic?

Reply:
dobra, to jeśli uważacie że teraz to jest bardzo źle to zmienie żeby name/message mogly byc nullami

Reply:
W sensie mi tutaj konkretnie chodzi o to że jeśli wywołamy `withName(null)` to albo to powinno resetować nazwę albo rzucać exception